src/utils/FetchAddClub.js

```diff
@@ -0,0 +1,21 @@
+const URL = 'http://localhost:8080/clubs';
```

Comment:
Generally when you're creating a front-end you'll want to create a "client" object that acts as an interface to your backend API. In our case the Bounce client would make the necessary calls to the backend. I've put an example up for you to copy from on the `client` branch here https://github.com/ubclaunchpad/bounce-front-end/blob/client/src/Client.js.